src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java

```diff
@@ -24,10 +26,12 @@
 import org.jenkins.tools.test.model.hook.PluginCompatTesterHookBeforeCheckout;
 
 /**
- * Utility class to ease create simple hooks for multimodule projects
+ * Utility class to ease create simple hooks for multi-module projects
```

Comment:
The Maven documentation (not that it is a particularly consistent source of correct spelling) spells this with a hyphen, so do the same here.